invenio_rdm_records/collections/services/service.py

```diff
@@ -131,3 +165,47 @@ def read_logo(self, identity, slug):
         if _exists:
             return url_for("static", filename=logo_path)
         raise LogoNotFoundError()
+
+    def read_many(self, identity, ids_, depth=2):
```

Comment:
Good point. 

IMO it's useful to have it alongside `read_all` to make a clear distinction since the underlying queries are distinct and have performance differences. 

Therefore I prefer it stay there from early on.